packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -278,6 +279,68 @@ const CFDsListing = () => {
             ) : (
                 <PlatformLoader />
             )}
+
+            {is_landing_company_loaded ? (
+                available_derivez_accounts?.map((account: AvailableAccount) => {
+                    const existing_accounts = getExistingAccounts(account.platform, account.market_type);
+                    const has_existing_accounts = existing_accounts.length > 0;
+                    return has_existing_accounts ? (
+                        existing_accounts.map((existing_account: TDetailsOfEachMT5Loginid) => (
+                            <TradingAppCard
+                                action_type='multi-action'
+                                availability={selected_region}
+                                clickable_icon
+                                icon={account.icon}
+                                sub_title={account.name}
+                                name={`${formatMoney(
+                                    existing_account.currency,
+                                    existing_account.display_balance,
+                                    true
+                                )} ${existing_account.currency}`}
+                                description={existing_account.display_login}
+                                platform={account.platform}
+                                key={`trading_app_card_${existing_account.display_login}`}
+                                onAction={(e?: React.MouseEvent<HTMLButtonElement>) => {
+                                    const button_name = e?.currentTarget?.name;
+                                    if (button_name === 'transfer-btn') {
+                                        toggleAccountTransferModal();
+                                        setSelectedAccount(existing_account);
+                                    } else if (button_name === 'topup-btn') {
+                                        showTopUpModal(existing_account);
+                                    } else {
+                                        startTrade(account.platform, existing_account);
+                                    }
+                                }}
+                            />
+                        ))
+                    ) : (
+                        <TradingAppCard
+                            action_type='get'
+                            availability={selected_region}
+                            clickable_icon
+                            icon={account.icon}
+                            name={account.name}
+                            platform={account.platform}
+                            description={account.description}
+                            onAction={() => {
+                                if ((has_no_real_account || no_CR_account) && is_real) {
```

Comment:
how can (has_no_real_account || no_CR_account) && is_real be true at the same time?

First condition evaluates if you dont have CR or real account.

is_real is whether or not active/selected account is real.

You cant select real account if you have no real account so one of the conditions will be false always.

Reply:
I'm gonna get clarified on the flags, 

I think the `is_real` flag checks if user trying to get a real or demo (say) derivez account

If they're trying to get a real account but haven't got a CR account yet

then `openDerivRealAccountNeededModal` to open a CR account before getting the derivez app 

@jim-deriv help 🫠 is this correct ..?

Reply:
Hi @shontzu-deriv and @henry-deriv, The `is_real` computed reflects the value selected in the  `dropdown` seen in the attached screenshot, which doesn't require you to have a `real` account to change, so if it's current value is `demo`, `is_real` will be `false`, whether or not you have a real account.

![dropdown](https://user-images.githubusercontent.com/104334373/234815119-b2a636fc-0e83-482b-962e-10c2f99a8b76.png)

`has_no_real_account` is the one that has to do with you having a `real` account or not

Reply:
ok in that case. If condition logic is fine.